public/data/NextThread.json

```
@@ -0,0 +1,21 @@
+{
+    "name": "Anurag Roy",
+    "location": "India",
+    "bio": "Finding & fixing BUG, Open Source dev, CPer, Mentor",
+    "avatar": "https://github.com/NextThread",
```

```suggestion
    "avatar": "https://github.com/NextThread.png",
```
add .png to avatar line.

---

@shyamtawli  done